.github/workflows/.test.yml

```diff
@@ -366,9 +446,43 @@ jobs:
           matrix="$(gotestlist -d ${{ env.ITG_CLI_MATRIX_SIZE }} -o "./..." -o "DockerSwarmSuite" -o "DockerNetworkSuite|DockerExternalVolumeSuite" ./...)"
           echo "matrix=$matrix" >> $GITHUB_OUTPUT
       -
-        name: Show matrix
+        name: Create gha matrix
+        id: set
+        uses: actions/github-script@v7
+        with:
+          script: |
+            let matrix = {
+              test: ${{ steps.tests.outputs.matrix }},
+              include: [],
+            };
+            // For some reasons, GHA doesn't combine a dynamically defined
+            // 'include' with other matrix variables that aren't part of the
+            // include items.
+            // Moreover, since the goal is to run only relevant tests with
+            // firewalld enabled to minimize the number of CI jobs, we
+            // statically define the list of test suites that we want to run.
+            if ("${{ inputs.storage }}" == "snapshotter") {
```

Comment:
`integration-cli` test suites are deprecated, and the `test/validate` GHA job executes `hack/validate/deprecate-integration-cli` which should error out if any new test is added to the `integration-cli` test suites. So I assume it's perfectly fine to define these suites statically.